Source/Diagnostics/FlushFormats/FlushFormatOpenPMD.H

```diff
@@ -3,9 +3,16 @@
 
 #include "FlushFormat.H"
 #include "Diagnostics/WarpXOpenPMD.H"
+// _______________________________________________WarpX Forward declarations
```

Comment:
I would not write such comment blocks, because we instead create a convention for all of them.

But you could add a sentence or two in the developer docs where we explain include order, etc. :)